README.md

```diff
@@ -35,18 +29,11 @@ reflective of the final deployment environment or AOT deployment. Supported
 runtimes:
 
 * JAX, Tensorflow, PyTorch and TFLite with IREE using MLIR input.
-* JAX, Tensorflow with XLA using HLO input.
 * TFLite.
 * GGML (experimental).
 
 ## Supported Devices
 
-### Server
-
-* GPU: a2-highgpu-1g.
-* CPU: c2-standard-60.
-* (Retired) c2-standard-16.
```

Comment:
we use these for server benchmarking, even just for TFLite vs IREE

Reply:
As of 3 weeks ago, these benchmarks still work on both GPU and CPU: https://github.com/openxla/iree-comparative-benchmark/actions/runs/8242578646.